airflow/providers/microsoft/azure/transfers/local_to_adls.py

```diff
@@ -100,3 +100,20 @@ def execute(self, context: dict) -> None:
             blocksize=self.blocksize,
             **self.extra_upload_options,
         )
+
+
+class LocalToAzureDataLakeStorageOperator(LocalFilesystemToADLSOperator):
```

Comment:
Can you add this class to file: https://github.com/apache/airflow/blob/main/tests/deprecated_classes.py?

Reply:
To my knowledge this file is used to test the classes migrated from contrib to providers. It's being used in 
https://github.com/apache/airflow/blob/8505d2f0a4524313e3eff7a4f16b9a9439c7a79f/tests/core/test_core_to_contrib.py#L29

The `LocalFilesystemToADLSOperator` wasn't part of contrib. It was added directly to providers in https://github.com/apache/airflow/pull/10814